### PR TITLE
Data class

### PR DESCRIPTION
**Pull request by nullArrayList**

### CODE REVIEW

File: app/src/main/kotlin/com/hippo/ehviewer/client/parser/GalleryDetailParser.kt [hunk 1]
```diff
@@ -76,7 +76,7 @@ object GalleryDetailParser {
     private val PATTERN_NEWER_DATE = Regex(", added (.+?)<br />")
     private val PATTERN_FAVORITE_SLOT =
         Regex("/fav.png\\); background-position:0px -(\\d+)px")
-    private val EMPTY_GALLERY_TAG_GROUP_ARRAY = arrayOf<GalleryTagGroup>()
+    private val EMPTY_GALLERY_TAG_GROUP_ARRAY = emptyList<GalleryTagGroup>()
     private val EMPTY_GALLERY_COMMENT_ARRAY = GalleryCommentList(emptyList(), false)
```
Comment (FooIbar) [inline app/src/main/kotlin/com/hippo/ehviewer/client/parser/GalleryDetailParser.kt:80]: Rename these two to match their types?

**Review by FooIbar — APPROVED** [review]

File: app/src/main/kotlin/com/hippo/ehviewer/client/parser/GalleryDetailParser.kt (hunk 1 above)
Comment (nullArrayList) [reply app/src/main/kotlin/com/hippo/ehviewer/client/parser/GalleryDetailParser.kt:79]: I inlined it since emptyList() always return a singleton

Comment (FooIbar) [reply app/src/main/kotlin/com/hippo/ehviewer/client/parser/GalleryDetailParser.kt:79]: The empty GalleryCommentList object?

Comment (nullArrayList) [reply app/src/main/kotlin/com/hippo/ehviewer/client/parser/GalleryDetailParser.kt:79]: EMPTY_GALLERY_TAG_GROUP_ARRAY

Comment (FooIbar) [reply app/src/main/kotlin/com/hippo/ehviewer/client/parser/GalleryDetailParser.kt:79]: Yeah I mean rename the GalleryCommentList object.

Comment (nullArrayList) [reply app/src/main/kotlin/com/hippo/ehviewer/client/parser/GalleryDetailParser.kt]: Done

**Pull request merged**